wgsl/index.bs

```diff
@@ -2907,33 +3285,31 @@ The declaration must appear at [=module scope=], and its [=scope=] is the entire
 
     | [=syntax/uint32=]
 
-    | [=syntax/vec2=] [=syntax/less_than=] [=syntax/type_decl=] [=syntax/greater_than=]
+    | [=syntax/vec2=] ( [=syntax/less_than=] [=syntax/type_decl=] [=syntax/greater_than=] ) ?
```

Comment:
The grammar wasn't updated to account for this case when the type tables were (#2305).